src/utils/openCloseComponent.ts

```diff
@@ -66,13 +66,11 @@ function transitionEnd(event: TransitionEvent): void {
  * @param component
  */
 export function connectOpenCloseComponent(component: OpenCloseComponent): void {
+  if (componentToTransitionListeners.has(component)) {
```

Comment:
I don't think you need this check here. We want it to proceed to disconnect and add new entry.